### ISSUE TITLE
WinUI3 not working using C++ example

### ISSUE THREAD

**Issue by tyejac** [MicrosoftDocs/windows-uwp#3788]
[Enter feedback here]
Encounter a lot of errors when create a blank C++ for WinUI using VS2022.
Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'Application' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	HelloWinUICppWinRT	C:\Users\tyeja\source\repos\HelloWinUICppWinRT\HelloWinUICppWinRT\App.xaml	6	
Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'ResourceDictionary' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	HelloWinUICppWinRT	C:\Users\tyeja\source\repos\HelloWinUICppWinRT\HelloWinUICppWinRT\App.xaml	7	
Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'Application' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	HelloWinUICppWinRT	C:\Users\tyeja\source\repos\HelloWinUICppWinRT\HelloWinUICppWinRT\App.xaml	1	
Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'Application' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	HelloWinUICppWinRT	C:\Users\tyeja\source\repos\HelloWinUICppWinRT\HelloWinUICppWinRT\App.xaml	6	

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: eb0978bf-4d67-556c-b6c6-87b360d446c8
* Version Independent ID: 6fe15723-a3a2-9a7c-1720-ef4194f5ed70
* Content: [A simple C++/WinRT Windows UI Library example - UWP applications](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/simple-winui-example?source=docs)
* Content Source: [windows-apps-src/cpp-and-winrt-apis/simple-winui-example.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/cpp-and-winrt-apis/simple-winui-example.md)
* Product: **uwp**
* Technology: **platform**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Thanks, @tyejac. Yes, it wasn't clear that the topic is about WinUI 2 and not WinUI 3. So I've published an update to the topic to make that clear. Only WinUI 2 is meant to work with this walkthrough, and not WinUI 3. And I've added more info to the topic (and links) about using WinUI 3 with a desktop app.

In addition to the new links I've added to the topic, [A Windows App SDK migration of the UWP Photo Editor sample app (C++/WinRT)](https://docs.microsoft.com/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/case-study-2) shows WinUI 3 in use with a C++/WinRT desktop app (ported from a UWP app).

Thanks!
-Steve
